presto-main/src/main/java/com/facebook/presto/execution/SqlStageExecution.java

```diff
@@ -333,6 +334,14 @@ public synchronized Duration getTotalCpuTime()
         return new Duration(millis, TimeUnit.MILLISECONDS);
     }
 
+    public synchronized DataSize getRawInputDataSize()
+    {
+        long datasize = getAllTasks().stream()
```

Comment:
Lets add this check at the top to ensure that we are only considering table scan nodes - 

```
if (planFragment.getTableScanSchedulingOrder().isEmpty()) {
    return new DataSize(0, BYTE);
}
```

Reply:
agree, we need to bypass when source is empty, fixed